packages/explorer/src/components/Utxos.tsx

```diff
@@ -32,7 +32,7 @@ const UTXODisplay = () => {
       showLoader(true)
       const response = await computer.wallet.restClient.getUtxos(addr)
       setUtxos(response)
-      setTotalAmount(response.reduce((total, unspent) => total + unspent.satoshis / 1e8, 0))
+      setTotalAmount(response.reduce((total, unspent) => total + Number(unspent.satoshis) / 1e8, 0))
```

Comment:
We should calculate the total amount in satoshis and use `formatBalance` in line 57 where we display `totalAmount`. We should also consider renaming `formatBalance` into `formatAmount` or `formatSatoshis`.